app/views/admin/users/_user_results.html.haml

```diff
@@ -0,0 +1,26 @@
+= simple_form_for :search, method: :get do |s|
+  = s.input :search_term, label: "User email search", placeholder: "type email here", input_html: { value: (params[:search][:search_term] unless params[:search].blank?) }
+  = s.button :submit, "SEARCH"
```

Comment:
Would prefer label were "Search". No need for it to be all caps and up in your face.